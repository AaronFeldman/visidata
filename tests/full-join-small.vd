sheet	col	row	keystrokes	input	comment
			o	sample_data/blah.tsv	open input in VisiData
			o	sample_data/blah2.tsv	open input in VisiData
blah2	Key	0	!		pin current column to the left as a key column
blah2	oneway	0	S		open Sheets Sheet
sheets	name	2	^J		jump to sheet referenced in current row
blah	Key	0	!		pin current column to the left as a key column
blah	A	0	S		open Sheets Sheet
sheets	name	1	s		select current row
sheets	name	2	s		select current row
sheets	name	2	&	full	merge the selected sheets with visible columns from all, keeping rows according to jointype
blah+blah2	A	0	!		pin current column to the left as a key column
blah+blah2	B	0	g[		sort ascending by all key columns
blah+blah2	A	0	!		pin current column to the left as a key column
