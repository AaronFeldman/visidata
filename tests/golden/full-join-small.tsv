Key	A	B	oneway	another
1	A	B		
2	C	D	F	H
2	G	F	F	H
3			Z	Y
